src/scripts/main.js

```diff
@@ -1,3 +1,376 @@
 'use strict';
 
-// write code here
+// Sort table
+const tHead = document.querySelector('thead');
+const tHeadItems = tHead.querySelectorAll('th');
+
+const tBody = document.querySelector('tbody');
+const trsBody = tBody.querySelectorAll('tr');
+
+let lastActiveTrBody;
+let sortedTrsBody;
+let isClickedFirst = false;
+
+tHeadItems.forEach((tHeadItem, index) => {
+  tHeadItem.addEventListener('click', () => {
+    isClickedFirst = !isClickedFirst;
+
+    if (isClickedFirst) {
```

Comment:
you're duplicating a big part of your code
try to overwrite it a bit (e.g. handle sort order in every separate case)